modules/ROOT/pages/mule-app-properties-to-configure.adoc

```diff
@@ -23,9 +23,14 @@ http.port=10000
 http.host=my-api.cloudhub.io
 ----
 
+Use the file format that makes it easier for you to write and read the properties and values.
+
 == Create and Configure your Properties File
 
-. Create a new `.yaml` or `.properties` file in your project, and store it in `/src/main/resources`.
+. In Studio, go to the *Package Explorer* view and right click the `/src/main/resources` folder in your project. Then, select *New* > *File*:
++
+image:mruntime-properties-new-file.png[new-file-properties]
+. Choose a name for your file and set the extension to either `.yaml` or `.properties`, depending on which format you prefer to use.
```

Comment:
as mentioned https://github.com/mulesoft/docs-mule-runtime/blob/v4.2/modules/ROOT/pages/configuring-properties.adoc#supported-files, maybe add the "disclaimer": 
> the recommended approach is to use a YAML configuration files